cloudbuild.yaml

```diff
@@ -374,6 +374,23 @@ steps:
     dir: cmd/sdk-server/bin
     args: [-m, cp, '*.zip', gs://agones-artifacts/sdk-server]
 
+  #
+  # Cleanup services "preview", "development", "default"
+  #
+  - name: gcr.io/cloud-builders/gcloud
+    id: cleanup-services
+    waitFor: [htmltest-save-cache, rust-build-save-cache, tests, build-images]
```

Comment:
You frankly don't need to wait for anything here, it just needs to run...

Reply:
Actually, if you wait for `e2e-wait-to-become-leader` here instead, it will mean that only one PR build will attempt to delete app versions at the same time, which might obviate my comment below.